src/briefcase/platforms/linux/system.py

```diff
@@ -0,0 +1,1003 @@
+                            f"Release:        {getattr(app, 'revision', 1)}%{{?dist}}",
+                            f"Summary:        {app.description}",
+                            "",
+                            f"License:        {getattr(app, 'license', 'Unknown')}",
+                            f"URL:            {app.url}",
+                            "Source0:        %{name}-%{version}.tar.gz",
+                            "",
+                        ]
+                        + [
+                            f"Requires:       {requirement}"
+                            for requirement in system_runtime_requires
+                        ]
+                        + [
+                            "",
+                            f"ExclusiveArch:  {self.tools.host_arch}",
+                            "",
+                            "%description",
+                            app.long_description,
+                            "",
+                            # Disable all the auto-detection that tries to magically
+                            # determine requirements from the binaries
+                            f"%global __requires_exclude /usr/lib/{app.app_name}",
+                            f"%global __provides_exclude /usr/lib/{app.app_name}",
+                            # Disable all build post-processing so that binary
+                            # wheels aren't munged.
+                            "%global _enable_debug_package 0",
+                            "%global debug_package %{nil}",
+                            "%global __os_install_post /usr/lib/rpm/brp-compress %{nil}",
+                            "",
+                            "%prep",
+                            "%autosetup",
+                            "",
+                            "%build",
+                            "",
+                            "%install",
+                            "cp -r usr %{buildroot}/usr",
+                        ]
+                    )
+                )
+
+                f.write("\n\n%files\n")
+                # Build the file manifest. Include any file that is found; also include
+                # any directory that includes an app_name component, as those paths
+                # will need to be cleaned up afterwards. Files that *aren't*
+                # in <app_name> (sub)directories (e.g., /usr/bin/<app_name> or
+                # /usr/share/man/man1/<app_name>.1.gz) will be included, but paths
+                # *not* cleaned up, as they're part of more general system structures.
+                for filename in sorted(self.project_path(app).glob("**/*")):
+                    path = filename.relative_to(self.project_path(app))
+
+                    if filename.is_dir():
+                        if app.app_name in path.parts:
+                            f.write(f"%dir /{path}\n")
+                    else:
+                        f.write(f"/{path}\n")
```

Comment:
`rpmbuild` provides the ability to [compile](https://fedoraproject.org/wiki/Changes/No_more_automagic_Python_bytecompilation) the Python source before packaging.

If this is desirable at some point, this can be added after the `%install` step:
```
f"%py_byte_compile %{{__python3}} %{{buildroot}}/%{{_libdir}}/{app.app_name}/app/{app.app_name}/",
```

However, this will require changing how the `%files` section is built to only include higher level directories so that the additional files are included.

Reply:
The packaged app *should* [have byte code disabled](https://github.com/beeware/briefcase-linux-system-template/blob/main/%7B%7B%20cookiecutter.format%20%7D%7D/bootstrap/main.c#L53), and we strip `__pycache__` files before packaging (because they include path elements that are sensitive to their compilation path, which confuses debug traces).